src/lib/ngsi9/RegisterContextRequest.cpp

```diff
@@ -82,8 +82,8 @@ std::string RegisterContextRequest::check(ConnectionInfo* ciP, RequestType reque
   {
     alarmMgr.badInput(clientIp, "empty contextRegistration list");
     response.errorCode.fill(SccBadRequest, "Empty Context Registration List");
-  }
-  else if (((res = contextRegistrationVector.check(ciP, RegisterContext, indent, predetectedError, counter)) != "OK") ||
+  } 
```

Comment:
As discussed at skype, let's consider style-check aspect in the last PR in the "refactor cycle". NTC